index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>JoseMontoya.com</title>
+    <link rel="stylesheet" href="./style.css">
+</head>
+<body>
+    <header>
+        <div class="status_bar">
+            <p>9:41</p>
+            <img src="./assets/status_bar_icons.svg" alt="phone icons">
+        </div>
+        <div class="tool_bar">
+            <h1>Jose Montoya</h1>
+            <img src="./assets/menu_icon.svg" alt="menu">
```

Comment:
- Kindly make sure the logo (if the design has one) is on the left and has a link (a tag). I can suggest you to use the div instead of the `h1 `tag so that you can use your h1 to tag the title `<h2>Hey There. <br/> I'm Jose</h2>`
- [ ] Great job using the `<img>` tag here for your hamburger menu  However I think it would be better to change to also put it in the ` <nav>` tag, The nav tag is for semantics reasons. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) for more info :+1: 

- [ ] Kindly make sure the logo (if the design has one) has a link (`a` tag).

* This is for three reasons: convention, semantics, and user experience(UX). [Check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info. The logo (if the design has one) is on the left and has a link (a tag). :+1: 

example 
```
<header class="header"> 
  <div class="logo"> <a href="#"><img src="./images/logo.png" alt="Logo"></a></div>
   
      <nav class="menu">
        <a href="#"><img src="./images/hamburger.png" alt="hamburger" class="hamburger"></a>
      </nav>
    </header>
```